### ISSUE TITLE
Toast fails with broken symlinks

### ISSUE THREAD

**Issue by jazzdan** [stepchowfun/toast#245]
**Description**
If the inputs to a task contain a broken symlink toast will fail with: 

```
[ERROR] Unable to canonicalize path /Users/dan/development/toast_issue_repo/./broken_ymlink. Reason: No such file or directory (os error 2)
```

**Instructions to reproduce the bug**
1. `git clone https://github.com/jazzdan/toast_broken_symlink_issue.git`
2. `cd toast_broken_symlink_issue`
3. `toast`

**System information:**
 - OS: macOS Mojave 10.14.5
 - Docker version:
```
Client: Docker Engine - Community
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        6247962
 Built:             Sun Feb 10 04:12:39 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 04:13:06 2019
  OS/Arch:          linux/amd64
  Experimental:     true
```
 - Toast version: 0.24.0

**Comment by jazzdan** [comment]
To be clear I would expect toast to copy the broken symlink in to the container as is and not try to do any canonicalization.

**Comment by stepchowfun** [comment]
Thank you for this bug report! I will take a look this weekend.

**Comment by stepchowfun** [comment]
Fixed by https://github.com/stepchowfun/toast/pull/247 and shipped in v0.25.0.
